corehq/apps/reports/tasks.py

```diff
@@ -183,8 +184,7 @@ def monthly_reports():
 
 @periodic_task(run_every=crontab(hour=[22], minute="0", day_of_week="*"), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE','celery'))
 def saved_exports():
-    for group_config in HQGroupExportConfiguration.view("groupexport/by_domain", reduce=False,
-                                                        include_docs=True).all():
+    for group_config in get_all_hq_group_export_configs:
```

Comment:
does it make sense to add a classmethod to wrap ```get_all_hq_group_export_configs``` and quickcache that too?  Since this code is only run once a day I'm guessing no.